imports/ui/JapaneseNewGame.js

```diff
@@ -1568,6 +1573,37 @@ function setAllGUIRiichisFalse() {
 	Session.set("north_riichi", false);
 };
 
+function resetRoundStats() {
+	Session.set("current_points", 0);
+	Session.set("current_fu", 0);
+	Session.set("current_dora", 0);
+	Session.set("round_winner", Constants.NO_PERSON);
+	Session.set("round_loser", Constants.NO_PERSON);
+	Session.set("round_pao_player", Constants.NO_PERSON);
+
+	for (let k in Constants.DIRECTIONS) {
+		let wind = Constants.DIRECTIONS[k];
+		Session.set(wind + "_riichi", false);
+		Session.set(wind + "_tenpai", false);
+	}
+
+	$( ".winner_buttons button" ).removeClass( "disabled" );
+	$( ".loser_buttons button" ).removeClass( "disabled" );
+	$( ".riichi_buttons button" ).removeClass( "disabled" );
+	$( ".tenpai_buttons button" ).removeClass( "disabled" );
+	$( ".pao_buttons button" ).removeClass( "disabled" );
+
+	$( ".winner_buttons button" ).removeClass( "active" );
+	$( ".loser_buttons button" ).removeClass( "active" );
+	$( ".riichi_buttons button" ).removeClass( "active" );
+	$( ".tenpai_buttons button" ).removeClass( "active" );
+	$( ".pao_buttons button" ).removeClass( "active" );
+
+	$( "select.points" ).val(undefined);
```

Comment:
Does it set the dropdowns do something?  It should set the dropdowns to the unselectable first entry.